### PR TITLE
Minor InvocationRegistry cleanup

### PR DESCRIPTION
**Pull request by pveentjer**
The calculation of the concurrency level is now moved into the InvocationRegistry. It
is no longer a concern of the OperationServiceImpl. 

This makes the OperationServiceImpl a bit cleaner since it doesn't need to know about implementation details of the InvocationRegistry.

I believe this is an called an improved cohesion since letting the operationservice take care of a calculation that isn't its primary concern is causing low cohesion. And by moving it to the InvocationRegistry it increased cohesion since the OperationServiceImpl doesn't do something it doesn't care about; and it is moved to a place where it is important.

### CODE REVIEW

File: hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OperationServiceImpl.java [hunk 1]
```diff
@@ -148,22 +145,17 @@ public OperationServiceImpl(NodeEngineImpl nodeEngine) {
         this.nodeEngine = nodeEngine;
         this.node = nodeEngine.getNode();
         this.thisAddress = node.getThisAddress();
-        this.logger = node.getLogger(OperationService.class);
+        this.logger = node.loggingService.getLogger(OperationService.class);
```
Comment (mmedenjak) [inline hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OperationServiceImpl.java]: `Node#getLogger(java.lang.Class)` will do the same thing, forward the method call to `LoggingServiceImpl#getLogger(java.lang.Class)`. Is this change deliberate then? Should other calls to `Node#getLogger(java.lang.Class)` in this class be changed too?

Comment (pveentjer) [inline hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OperationServiceImpl.java]: I believe it should be changed. Node isn't responsible for providing an API to create a logger; that is already the concern of the LoggingService.

Comment (pveentjer) [inline hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OperationServiceImpl.java]: I'll remove it from this PR so it remains clean.

**Comment by pveentjer** [comment]
Minor comment regarding logger has been resolved by reverting the log chance. So merging this one
